server.js

```diff
@@ -35,6 +35,31 @@ app.post('/items', function(req, res) {
   });
 });
 
+app.put('/items/:id', function(req, res) {
+  let id = req.params.id;
+  let newName = req.body.name;
+
+  if (id === 'undefined') res.status(400).send('Bad request, invalid ID');
+  if (newName === 'undefined' || newName === null) res.status(400).send('Bad request, no name given'); //This isn't working yet
```

Comment:
^ I like this philosophy @amcavinue. I was getting a little lazy last night at commit :sleeping: